logstash/postgres_to_elasticsearch.conf

```diff
@@ -0,0 +1,90 @@
+input {
+  jdbc {
+    jdbc_driver_library => "/usr/share/logstash/postgresql-42.7.3.jar"
+    jdbc_driver_class => "org.postgresql.Driver"
+    jdbc_connection_string => "jdbc:postgresql://db:5432/architecture-map?currentSchema=architecture_map"
+    jdbc_user => "${POSTGRES_USER}"
+    jdbc_password => "${POSTGRES_PASSWORD}"
+    schedule => "0 6 * * * America/Chicago"
```

Comment:
why America/Chicago?

Reply:
It is not really important in which time we will run script, the main that it runs every day. So, I was go the simplest way and used time from documentation :)

https://www.elastic.co/guide/en/logstash/current/plugins-inputs-jdbc.html